assets/i18n/en.json

```diff
@@ -519,16 +519,41 @@
     "I18N_SPLASH_FIRST_EXPLORATION_DESCRIPTION": "Oppia's lessons, also known as explorations, provide more immersive experiences than static videos or text, helping users learn by doing.",
+    "I18N_SPLASH_FOR_STUDENTS": "For Students",
+    "I18N_SPLASH_FOR_TEACHERS": "For Teachers",
+    "I18N_SPLASH_FOR_VOLUNTEERS": "For Volunteers",
+    "I18N_SPLASH_ICON_ONE_TEXT": "1 Million+ Users",
+    "I18N_SPLASH_ICON_TWO_TEXT": "Available in 4+ Languages",
+    "I18N_SPLASH_ICON_THREE_TEXT": "40 Curated Lessons",
     "I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION": "Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href=\"<[hrefUrl]>\">click here.</a>",
     "I18N_SPLASH_JAVASCRIPT_ERROR_THANKS": "Thank you.",
     "I18N_SPLASH_JAVASCRIPT_ERROR_TITLE": "We Need JavaScript in Your Browser",
+    "I18N_SPLASH_LEARN_MORE": "Learn More",
     "I18N_SPLASH_PAGE_TITLE": "Oppia | Free, Online and Interactive Lessons for Anyone",
     "I18N_SPLASH_SECOND_EXPLORATION_DESCRIPTION": "Explorations are simple to create. They are easily adapted based on individual student feedback and trends in the experience of your learners around the world.",
     "I18N_SPLASH_SITE_FEEDBACK": "Site Feedback",
-    "I18N_SPLASH_SUBTITLE": "Oppia makes it easy to create interactive lessons that educate and engage.",
+    "I18N_SPLASH_START_CONTRIBUTING": "Start Contributing",
+    "I18N_SPLASH_STUDENT_DETAILS_1": "- Mira, Student, Palestine",
+    "I18N_SPLASH_STUDENT_DETAILS_2": "- Sama, Student, Palestine",
+    "I18N_SPLASH_STUDENT_DETAILS_3": "- Sara, Student, Palestine",
+    "I18N_SPLASH_STUDENTS_CONTENT": "Oppia identifies common wrong answers and provides tailored feedback, so that students get a personalized experience. Our lessons retain engagement through playful characters that lead students and diverse strategies that solidify knowledge. Check out our math lessons with proven results!",
+    "I18N_SPLASH_STUDENTS_TITLE": "An Immersive Environment",
+    "I18N_SPLASH_SUBTITLE": "Engaging, effective, personalized",
+    "I18N_SPLASH_TEACHERS_CONTENT": "With Oppia's content creation system, you can easily customize the lesson's format for your students. Create and share lessons on topics you are passionate about.",
+    "I18N_SPLASH_TEACHERS_TITLE": "Easily Share Your Knowledge",
+    "I18N_SPLASH_TESTIMONIAL_1": "\"I enjoyed playing the lesson a lot, I did not feel bored and I feel like I master the negative numbers now\"",
+    "I18N_SPLASH_TESTIMONIAL_2": "\"I learned ratios at school but with Oppia I introduced to many new things like the three items ratios and combining ratios\"",
+    "I18N_SPLASH_TESTIMONIAL_3": "\"I love Oppia so much, and I hope that more topics like geographic  and science are available in Arabic\"",
```

Comment:
geographic --> geography

Drop extra space before "and".

Reply:
Done